### DISCUSSION TITLE
Pull function does not allow me to store the score grid with pycaret 3.0.0.rc2 time series

### DISCUSSION THREAD

**Discussion by acartro** [pycaret/pycaret#2725] [Q&A] [answered]
Good morning,

I have already seen that there are discussions on how to get the metrics using pull. But in my case, I am with pycaret 3.0.0.rc2, and I can't get the cross validation metrics using pull.

Does anyone know why or how I can go inside create_models and get the error metrics?

```
from pycaret.time_series import *
from pycaret.datasets import get_data
from sktime.transformations.series.date import DateTimeFeatures

# get data
y = get_data("airline")
#### Setup experiment ----
exp = TSForecastingExperiment()
exp.setup(data=y, fh=12, session_id=42, verbose=False)

# create model
model = exp.create_model("arima")
# get metrics
dt_results = pull()

```

![image](https://user-images.githubusercontent.com/11137446/177723864-458eeec8-0096-491e-9488-de1596143e88.png)

**Comment by ngupta23** [comment, marked as answer]
I think since you are using the OOP interface, you need to do `exp.pull()`.

**Reply by acartro** [reply]
Yes, it is true.
Thank you very much.
